packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -67,6 +68,51 @@ test.describe('Visual Editing', () => {
     await sendContentUpdateMessage({ page, newContent: MODIFIED_COLUMNS, model: 'page' });
     await page.frameLocator('iframe').getByText(NEW_TEXT).waitFor();
   });
+
+  test('correctly updates Box -> Columns when used Inner Layout > Columns option', async ({
+    page,
+    packageName,
+    basePort,
+  }) => {
+    test.skip(
+      packageName === 'react-native' ||
```

Comment:
Remind me why we're skipping RN SDK? I think this is an old issue before we started building the e2e RN app properly. I wonder if we no longer need to skip it anymore.

If you try and it works, i would remove this. If not, would be good to put it in a separate `test.skip()` and explain briefly why it's failing (if you're able to see why locally)

Reply:
ahh right, for this specific test, we will have the same layout (stacked) for columns in RN. Have updated the tests to include React native and they all are passing